SEBrowser/Controllers/OpenXDAController.cs

```diff
@@ -488,12 +491,11 @@ Asset  JOIN
 	                    (SELECT ROUND(MAX(IMax),3) as IMax FROM (VALUES(IAMax), (IBMax), (ICMax)) AS IMaxView(IMax)) as IMax OUTER APPLY
 	                    (SELECT ROUND(MAX(I2tMax),3) as I2tMax FROM (VALUES(IA2t), (IB2t), (IC2t)) AS I2tView(I2tMax)) as I2tMax
                     WHERE Asset.ID = (SELECT AssetID FROM Event WHERE ID = {0})
-                    GROUP BY VoltageKV
+                    GROUP BY VoltageKV , Asset.AssetName

```

Comment:
```suggestion
                    GROUP BY Asset.AssetName

```

Reply:
Making this change causes the following error: System.Data.SqlClient.SqlException: 'Column 'Asset.VoltageKV' is invalid in the select list because it is not contained in either an aggregate function or the GROUP BY clause.'